.vscode/tasks.json

```diff
@@ -97,6 +97,18 @@
         "problemMatcher": "$msCompile",
         "group": "build"
     },
+    {
+        "label": "build toolset",
+        "type": "shell",
+        "command": "dotnet",
+        "args": [
+          "build",
+          "-p:GenerateFullPaths=true",
+          "src/NuGet/Microsoft.Net.Compilers.Toolset/AnyCpu/Microsoft.Net.Compilers.Toolset.Package.csproj"
```

Comment:
Most task args prefix with something like `${workspaceFolder}`, do we need to do that here also for things to be correct?